### PR TITLE
[SMALLFIX]Added a `worker Id` table column to the workers page.

### PR DESCRIPTION
**Pull request by maobaolong**
Without worker id, we don't konw the workerid in the `master.log` means which worker.
Now we can find worker hostname from worker id in the workers page.

### CODE REVIEW

**Comment by maobaolong** [comment]
alluxio-bot, check this please

**Comment by yupeng9** [comment]
@maobaolong can you explain how the worker id is useful in your use case?

**Comment by maobaolong** [comment]
@yupeng9 , After i execute `cat master.log | grep "worker id"`, i find  `Could not find worker id: 6434053585924026815 for heartbeat.`, and i don't know what the `worker id` stand for.

**Comment by maobaolong** [comment]
Reference the class `alluxio.master.block.BlockMaster`, you can find many log output the workerId.

I execute `cat master.log* | grep "workerId", i find the following:
```
2017-03-12 16:16:01,658 INFO  logger.type (MasterWorkerInfo.java:register) - re-registering an existing workerId: 6434053585924026815

2017-03-08 15:30:37,647 WARN  logger.type (BlockMaster.java:processWorkerAddedBlocks) - Failed to register workerId: 8660057149119863947 to blockId: 1034315366400

2017-03-12 14:49:04,949 WARN  logger.type (BlockMaster.java:processWorkerAddedBlocks) - Failed to register workerId: 1354440546517865611 to blockId: 3313131061248

```

Through the `workerId` is really hard to guess which worker it stand for.

**Comment by yupeng9** [comment]
I see. So the purpose is to understand the logs for better debugging/monitoring. @jsimsa what do you think?

**Comment by calvinjia** [comment]
I would prefer to not expose worker id to the user and make the logs more informative.

**Comment by jsimsa** [comment]
@calvinjia any suggestion on how to make the logs more informative without exposing the worker / block IDs?

**Comment by calvinjia** [comment]
I haven't checked all the call sites, but I think we can avoid logging worker id and always use worker hostname or even the entire `MasterWorkerInfo`. 

It depends what we want to expose to the user. For example, some other ids we currently expose are block ids and file ids, but we may be able to ensure that ids are only used internally in the system and should not need to be interpreted by a user. In the case of worker id, I don't think there is any reason to expose this (ie. for block id it maps to the tiered storage file name so that might be useful).

**Comment by yupeng9** [comment]
@calvinjia is hostname always sufficient as the unique identifier of a worker? Is it possible to launch multiple workers on the same node?

But I agree with you on not exposing extra information unless necessary. From that perspective, @maobaolong if you are interested, do you want to replace the worker id in the logs to `MasterWorkerInfo`?

**Comment by maobaolong** [comment]
@yupeng9 @calvinjia @jsimsa in some place, it is hard to get workerinfo, it just have a worker id. if i output more log, i must change many functions signature. if i show the worker id when alluxio in debug mode, is this a way to do so?

**Comment by calvinjia** [comment]
@maobaolong If we need to display worker id, I prefer to see it only in debug mode.

**Comment by maobaolong** [comment]
@calvinjia PLAL.

**Comment by maobaolong** [comment]
@calvinjia i think this failed test is not relate to me.

**Comment by calvinjia** [comment]
Jenkins, test this please

File: core/server/common/src/main/webapp/workers.jsp [hunk 1]
```diff
@@ -92,6 +94,7 @@
               <thead>
                 <th>Node Name</th>
                 <% if ((Boolean) request.getAttribute("debug")) { %>
+                  <th>Worker Id</th>
```
Comment (calvinjia) [inline core/server/common/src/main/webapp/workers.jsp]: Could you add a `[D]` to be consistent with the other debug only fields?

Comment (maobaolong) [inline core/server/common/src/main/webapp/workers.jsp]: make sense.

**Comment by maobaolong** [comment]
@calvinjia  Done. PTAL.

**Pull request merged**